query-coordinator-http/src/main/resources/reference.conf

```diff
@@ -58,6 +58,7 @@ com.socrata.query-coordinator {
   get-schema-timeout = 10s
   query-timeout = 2m
   all-secondary-instance-names = [pg.primus]
+  secondary-selector-map-max-capacity = 2000
```

Comment:
This should be much bigger... the memory footprint of each entry is quite tiny ... without doing the math I'd guess that hundreds of thousands of entries should be only a few hundred megs, which is not much memory.